inventories/group_vars/all

```diff
@@ -344,6 +344,8 @@ grouper:
         - server_name: "{{ nginx_ssl.server_name }}"
           return: "https://$host$request_uri"
   loader:
+    compose_service: grouper_loader
+    container_name: grouper-loader
```

Comment:
Since these configs aren't actually used anywhere in the public ansible playbooks or roles, I'm not sure they belong in this `group_vars` file.
Should we move them to our internal ansible repo? It seems like this might be extra clutter for any 3rd parties deploying their own DE, since changing this has no effect except in our internal scripts.